configure.ac

```diff
@@ -466,10 +466,8 @@ AC_ARG_ENABLE([coverage],
 AC_ARG_ENABLE([slow-tests],
   [AS_HELP_STRING([--enable-slow-tests],[build and run slow tests])])
 
-if test x"$with_check" = xyes; then
 AC_ARG_ENABLE([syslog-tests],
   [AS_HELP_STRING([--enable-syslog-tests],[build and run syslog tests])])
```

Comment:
Can you advise as to whether this conditional is sensible, please?
My original intention was to lay an egg as carefully as possible because
libtool warns that `foo_LDFLAGS = -module` is not portable.

Any idea of having this portability tested in some minimalistic way
and turning "syslog-tests" on automatically if positive?

Or is it more a spurious warning of libtool?